gui/frontend/src/dashboard/dashboard.coffee

```diff
@@ -161,6 +161,7 @@ angular.module "mindbenderApp.dashboard", [
     $scope.title = "Snapshot " + $routeParams.snapshotId
     $scope.loading = false
     $scope.hideLoader = true
+    $scope.tableTab = { active: false }
```

Comment:
I see.  That makes sense.  Adding a short comment next to it may be useful in the future :)